package.json

```diff
@@ -1,148 +1,112 @@
 {
+  "private": true,
   "name": "redoc",
-  "description": "Swagger-generated API Reference Documentation",
-  "version": "1.19.1",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/Rebilly/ReDoc"
-  },
-  "engines": {
-    "node": ">=6.9",
-    "npm": ">=3.0.0"
-  },
-  "main": "dist/redoc.module.js",
-  "module": "dist/redoc.module.js",
-  "types": "dist/redoc.module.d.ts",
+  "version": "2.0.0",
+  "description": "ReDoc",
+  "main": "lib/index.js",
   "scripts": {
-    "start":
-      "webpack-dev-server --config build/webpack.dev.js --content-base demo",
-    "start:prod": "NODE_ENV=production npm start",
-    "test": "npm run lint && node ./build/run_tests.js",
-    "lint": "tslint -e \"lib/**/*{ngfactory|css.shim}.ts\" lib/**/*.ts",
-    "unit": "karma start",
-    "pree2e": "npm run build:prod && npm run e2e-copy",
-    "e2e": "run-p -r protractor e2e-server",
-    "protractor": "protractor",
-    "preprotractor": "npm run webdriver",
-    "e2e-server": "http-server -p 3000 tests/e2e",
-    "e2e-copy": "cp dist/redoc.min.js tests/e2e/",
-    "webdriver": "webdriver-manager update",
-    "deploy":
-      "node ./build/prepare_deploy.js && deploy-to-gh-pages --update demo",
-    "branch-release": "git reset --hard && branch-release",
-    "clean": "rimraf dist .tmp compiled lib/**/*.css",
-    "ngc": "ngc -p tsconfig.json",
-    "inline": "ng2-inline -o .tmp -r --compress \"lib/**/*.ts\"",
-    "build:module":
-      "npm run build:sass && npm run inline && ngc -p tsconfig.aot.json && npm run module:css",
-    "module:css": "node build/join-module-css.js",
-    "webpack:prod": "webpack --config build/webpack.prod.js --profile --bail",
-    "build:sass": "node-sass -q -o lib lib",
-    "build:prod": "npm run build:sass && npm run ngc && npm run webpack:prod",
-    "build-dist": "npm run build:prod",
-    "build:all": "npm run clean && npm run build:prod && npm run build:module",
-    "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s -r 1",
-    "stats": "webpack --config build/webpack.prod.js --json > stats.json"
+    "start": "webpack-dev-server --hot",
+    "start:perf": "webpack-dev-server --env.prod --env.perf",
+    "start:prod": "webpack-dev-server --env.prod",
+    "dev": "webpack-dashboard -- webpack-dev-server --hot",
+    "test": "jest",
+    "bundle": "webpack",
+    "bundle:prod": "webpack -p -env.prod",
```

Comment:
"bundle:prod": "webpack -p -env.prod" should be "bundle:prod": "webpack -p --env.prod",

Reply:
Yes, I've noticed the same issue but haven't tried to fix it yet. Just CTRL+C worked just fine for me (bundle actually is built and emitted to dist folder, but webpack process is not terminated by some strange reason).

Would be great if you could investigate this!

Reply:
It hangs there trying to connect 

`engine.io-client:polling-xhr xhr open GET: http://127.0.0.1:9838/socket.io/?EIO=3&transport=polling&t=LyeIbN8&b64=1 +1ms`

9838 is used by webpack-dashboard. But removing it from webpack.config.js/plugins, build process can complete.

Reply:
@shenghu thanks!